main/lsp/LSPMessage.cc

```diff
@@ -167,6 +167,8 @@ bool LSPMessage::isDelayable() const {
         case LSPMethod::TextDocumentPublishDiagnostics:
         case LSPMethod::SorbetShowOperation:
         case LSPMethod::SorbetTypecheckRunInfo:
+        // No-op message used to determine if the server has processed a message in the queue.
+        case LSPMethod::SorbetNop:
```

Comment:
what is it used for?

Reply:
It should be removed. From a previous and quickly abandoned approach. I opened this WIP immediately before cleaning it up.